@xen-orchestra/lite/src/views/new-vm/NewVmView.vue

```diff
@@ -472,7 +481,7 @@ const vmState = reactive<VmState>({
   vCPU: 0,
   VCPUs_max: 0,
   ram: 0,
-  topology: '',
+  topology: null,
```

Comment:
Don't forget to update your `onTemplateChange` to handle `topology` based on the `template.platform['cores-per-socket']`